tests/integration/docker-compose.yml

```
@@ -43,7 +43,7 @@ services:
     image: minio/minio:latest
     command: server /data
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
+      test: [ "CMD", "bash", "-c", ":> /dev/tcp/127.0.0.1/9000" ]
```

when formatting the YAML with the extension from our dev container, this is what it looks like:
```suggestion
      test: ["CMD", "bash", "-c", ":> /dev/tcp/127.0.0.1/9000"]
```

---

hm, ok! Ill try to find something for Goland

---

actually the same goes for the healthcheck you added a few days ago on line 60 — it should look like this:
```yaml
      test:
        ["CMD", "wget", "-q", "--spider", "http://localhost:4443/storage/v1/b"]
```

---

yes. fixed already.

---

that's a bit strange but it was auto formatting  in Goland but in just different style.